packages/bpk-component-modal/src/BpkModalDialog.js

```diff
@@ -44,7 +44,7 @@ export type Props = {
   className: ?string,
   contentClassName: ?string,
   title: ?string,
-  closeLabel: ?string,
+  closeLabel: string,
```

Comment:
As `bpk-component-close-button` has the label as required in the component changing this so that flow does not throw and error that null or undefined is not valid